modules/nw-route-specific-annotations.adoc

```diff
@@ -39,9 +39,10 @@ Note: Using this annotation provides basic protection against distributed denial
 |`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` | You can set either an IngressController or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop. | `ROUTER_HARD_STOP_AFTER`
 |`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`haproxy.router.openshift.io/ip_whitelist`
-| Sets a whitelist for the route. The whitelist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the whitelist are dropped.
+| Sets an allowlist for the route. The allowlist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the allowlist are dropped.
+
+The maximum number of IP addresses and CIDR ranges in an allowlist that can be written to `haproxy.config` is 64. [^1^] |
```

Comment:
i tested in 4.15 build the same and the whitelist which will be shown is 61 only and not 64. i would prefer the wordings of the sentence as `The maximum number of IP addresses and CIDR ranges directly visible in the`haproxy.config` is 61.`